packages/react/src/details/Details.tsx

```diff
@@ -32,7 +32,7 @@ export const Details = forwardRef<HTMLDetailsElement, DetailsProps>(
 			>
 				<Flex
 					as="summary"
-					inline
+					css={{ width: 'fit-content' }}
```

Comment:
Is it worth adding a comment what this does?

Reply:
- [ ] code comment the iOS announcement reasoning